packages/app/App.tsx

```diff
@@ -4,7 +4,8 @@ import 'react-native-gesture-handler';
 import { ThemeProvider } from '@mf/ui/theme';
 import AsyncStorage from '@react-native-async-storage/async-storage';
 import { NavigationContainer } from '@react-navigation/native';
-import { withWalletConnect } from '@walletconnect/react-native-dapp';
+import WalletConnectProvider from '@walletconnect/react-native-dapp';
```

Comment:
I'll roll this provider up into an interface 'MFProvider' that can interface the calls for us, wondering if we should add redux too but we can discuss that in discord.